### ISSUE TITLE
Does not work on 7zipped files, needs to be extracted to work.

### ISSUE THREAD

**Issue by Elijah067** [andrebrait/1g1r-romset-generator#33]
I tried to use this on my DS no-intro set which is 7zipped but the generator says that `WARNING: no eligible candidates for <rom name> have been found!`. I tried to extract the roms and the generator works fine. 

I also tried --no-scan on the 7zipped rom files and it also does not work. Maybe due to the file extensions.

**Comment by andrebrait** [comment]
Indeed, it cannot check the contents of 7-zip archives. That is already implemented in DATROMTool and it does work, but it won't be implemented in 1g1r-romset-generator.

If you want to use `--no-scan` with compressed files, you need to add `-e 7z` so it knows it can match `.7z` files by name. Otherwise, it'll look only for `.nds` files (or whatever happens to be the extension used in the DAT you're using).

**Comment by jvook** [comment]
ROMSorter will convert 7z and rar rom collections to standard zip so they work with this tool
https://github.com/drakewill-CRL/ROMSorter

**Comment by andrebrait** [comment]
@jvook just a FYI, I have something in the works for that as well :)
